apps/chatbot/src/modules/chatbot.py

```diff
@@ -383,7 +383,7 @@ def chat_generate(
                     context += f"URL: {url}\n\n{node.text}\n\n------------------\n\n"
 
             except Exception as e:
-                response_str = "Scusa, non posso elaborare la tua richiesta.\nProva a chierdimi una nuova domanda."
+                response_str = "Scusa, non posso elaborare la tua richiesta.\nProva a chiedermi una nuova domanda."
```

Comment:
If you agree, I would change it to "formulare"
[source](https://www.treccani.it/magazine/lingua_italiana/domande_e_risposte/grammatica/grammatica_663.html)

```suggestion
                response_str = "Scusa, non posso elaborare la tua richiesta.\nProva a formulare una nuova domanda."
```